Movie_MVP/Movie_MVP/MovieListModule/Model/Genres.swift

```diff
@@ -0,0 +1,18 @@
+// Genres.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import SwiftyJSON
+
+// Жанры фильмов
+struct Genres {
+    /// Идентификатор
+    let id: Int?
+    /// Имя актера
+    let name: String?
+
+    init(json: JSON) {
+        id = json["id"].intValue
```

Comment:
```suggestion
        id = json["id"].int
```
если опциональный то без Value